documentation/Ingress-controller-upgrade.md

```diff
@@ -35,5 +35,6 @@ make test terraform-apply CONFIRM_TEST=yes
 
 
         ```
+- After successful upgrades for each environment, check metrics are still displaying and working well in Grafana dashboards: https://grafana.test.teacherservices.cloud, https://grafana.platform-test.teacherservices.cloud, https://grafana.teacherservices.cloud
```

Comment:
Do you mind changing the order? It would make sense to have platform-test, test, production
Also add development: https://grafana.clusterX.development.teacherservices.cloud